scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,255 @@ output:
+
+
+```{r celltype subdiagnosis UMAP}
+# # UMAP of all libraries, separating by diagnosis group and labeling cell type
 scater::plotReducedDim(integrated_sce,
                        dimred = "fastmnn_UMAP",
+                       # color each point by cell type
                        colour_by = "celltype_broad",
                        point_size= 0.5, 
                        point_alpha = 0.4,
                        other_fields = "diagnosis_group") +
+  # split plots by diagnosis group 
   facet_wrap(~ diagnosis_group)
 ```
-We can also look at the distribution of cell types across each sample.
 
-```{r}
+As expected, we see that the visual clusters of cell types are being separated most likely due to different subdiagnoses.
+Although we expect that the same cell types are present in both, each of the subdiagnoses for rhabdomyosarcoma have a distinct genomic and epigenomic landscape, likely accounting for the global gene expression changes seen between samples from these two subdiagnoses in the UMAP ([Shern _et al._ 2014](https://doi.org/10.1158/2159-8290.CD-13-0639), [Stewart _et al._](https://doi.org/10.1016/j.ccell.2018.07.012)).
+
+We can also use a stacked barplot to look at the distribution of cell types across each sample.
+
+```{r celltype barplot}
+# filter coldata data.frame to only include tumor cells
 tumor_cells_df <- coldata_df %>%
   dplyr::filter(stringr::str_detect(celltype_broad, "Tumor"))
 
+# create a stacked barplot
 ggplot(tumor_cells_df, aes(x = sample, fill = celltype_broad)) + 
     geom_bar(color = "black", size = 0.2) +
     labs(
-      x = "Sample",
+      x = "Library",
       y = "Number of cells", 
       fill = "Cell type"
     ) +
   theme_bw() +
   theme(axis.text.x = element_text(angle = 90))+
+  # split by diagnosis group 
+  # only show non-NA values on x-axis
   facet_wrap(~diagnosis_group, scales = "free_x")
 ```
 
-*Need to explain the difference between tumor/tumor_myoblast, etc*
-These plots show that ARMS and ERMS share a lot of the same cell types, but perhaps there are differences in the cells between subdiagnoses.
-We also see that 6 of the libraries show a mixture of tumor cells belonging to the mesoderm, myoblast, and myocyte classes.
+Similar to the UMAP, this plot shows that ARMS and ERMS share a lot of the same cell types, although it is likely that these cell types may have distinct gene expression patterns defined by the associated subdiagnosis.
+
+We also see that only 6 of these libraries have tumor cells that have been further classified into cell type. 
 3 libraries contain cells that are only classified as tumor or non-tumor, and tumor cells are not further classified and the remaining library appears to not have been assigned any cell types, as they are all labeled with `NA`.
-We will not be using these 3 libraries.
+We will continue our analysis only using the 6 libraries with fully classified cell types, removing the other 3 before we proceed with differential expression.
```

Comment:
If we really want to shorten this notebook, especially the "preamble" before actually getting into DE, we could consider just starting off with those 6 libraries from the get-go? But, I do like the exploration and I think it's important to see how these kinds of analysis sausages get made. 
Maybe DE is just our longest day!